config/routes.rb

```diff
@@ -6,6 +6,12 @@ def self.matches?(request)
   end
 end
 
+class ClaimsOnlyConstraint
+  def self.matches?(request)
+    HostingEnvironment.current_service(request) == :claims
+  end
+end
```

Comment:
How do you feel about moving this out into it's own file? I know we haven't done so elsewhere but I think it'd be nice to have a more structured system for our route constraints